src/libs/shouldUseStagingServer/index.native.js

```diff
@@ -0,0 +1,3 @@
+export default function shouldUseStagingServer(stagingServerToggleState) {
```

Comment:
Missing jsDoc. 
```suggestion
/*
 * @param {Boolean} stagingServerToggleState
 * @returns {Boolean}
 */
export default function shouldUseStagingServer(stagingServerToggleState) {
```

Reply:
Wasnt sure as it does not add much value in this case. Accepted, thanks!

Reply:
Normally, I suggest jSDoc for all methods/functions that have params.